src/prototypecomponentsnavigatorwindow.h

```diff
@@ -0,0 +1,44 @@
+#ifndef PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+#define PROTOTYPECOMPONENTSNAVIGATORWINDOW_H
+
+#include <QWidget>
+
+class QItemSelectionModel;
+class QAbstractItemModel;
+class QStandardItem;
+class QItemSelection;
+
+
+namespace Ui {
+class PrototypeComponentsNavigatorWindow;
+}
+
+class PrototypeComponentsNavigatorWindow : public QWidget
+{
+    Q_OBJECT
+
+public:
+    explicit PrototypeComponentsNavigatorWindow(QWidget *parent = nullptr);
+    ~PrototypeComponentsNavigatorWindow();
+
+public:
+    void setModel(QAbstractItemModel*);
+
+private:
+    Ui::PrototypeComponentsNavigatorWindow *ui;
+
+
```

Comment:
Remove extra blank lines in the entire file. It is enough one blank line to separate code.

Reply:
done